packages/flutter_tools/lib/src/asset.dart

```diff
@@ -313,6 +314,49 @@ DevFSContent _createFontManifest(Map<String, dynamic> manifestDescriptor,
   return new DevFSStringContent(JSON.encode(fonts));
 }
 
+// Given an assets directory like this:
+//
+// assets/foo
+// assets/var1/foo
+// assets/var2/foo
+// assets/bar
+//
+// variantsFor('assets', 'foo') => ['/assets/var1/foo', '/assets/var2/foo']
```

Comment:
variantsFor takes a single string, but you're passing two here, did you mean 'assets/foo'?